docs/src/components/pricing/WhatToExpect.tsx

```diff
@@ -1,67 +1,114 @@
-                  sx={{ ml: 1 }}
-                >
-                  Perpetual license in production
-                </Typography>
-              </Box>
-              <Typography variant="body2" color="text.secondary">
-                With your purchase, you are granted a license to use a version of the product
-                forever in a production environment. There are no further charges unless you choose
-                to renew support and maintenance to cover newer versions. Please note that while the
-                use of the software is perpetual in production, it&apos;s not perpetual for
-                development, so if you&apos;d like to keep making changes to your product,
-                you&apos;ll need to renew your license. We roll bug fixes, performance enhancements,
-                and other improvements into new releases; we don&apos;t patch, fix, or in any way
-                alter older versions.
+        <Grid item xs={12} sm={6}>
+          <Paper variant="outlined" sx={{ p: 2, height: '100%' }}>
+            <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
+              <ReplayRoundedIcon fontSize="small" color="primary" />
+              <Typography
+                fontWeight="bold"
+                component="h3"
+                color="text.primary"
+                variant="body2"
+                sx={{ ml: 1 }}
+              >
+                Development license
               </Typography>
-            </Paper>
-            <Paper variant="outlined" sx={{ p: 2, height: '100%' }}>
-              <Box sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
-                <HelpOutlineOutlinedIcon fontSize="small" color="primary" />
-                <Typography
-                  fontWeight="bold"
-                  component="h3"
-                  color="text.primary"
-                  variant="body2"
-                  sx={{ ml: 1 }}
-                >
-                  Support and maintenance
-                </Typography>
-              </Box>
-              <Typography variant="body2" color="text.secondary">
-                With your purchase, you receive support and maintenance for one year. After this
-                time, you can continue to use your licensed versions in production, but will no
-                longer be licensed to use the product in development, for instance, to release bug
-                fixes.
+            </Box>
+            <Typography variant="body2" color="text.secondary">
+              Developers contributing changes to the front-end code of a project that include the
+              software <strong>always</strong> need an active license. So if you wish to continue
+              active development using the software after the end of your subscription, you will
+              need to renew your subscription. You can learn more about it in{' '}
+              <Link href="https://mui.com/store/legal/mui-x-eula/">the EULA</Link>.
```

Comment:
I think it's nice we have it, but I was seeing this section more as a spot to advertise some advantages of using MUI X. 

Like: 
- Continuous delivery of improvements and new features.
- Effort to reach the best UX and DX.
- Community driven (we listen the community pains)

Perhaps we can merge this section with "Perpetual in production" and use the free slot to expose some attractive outcomes of our efforts.

WDYT?

Reply:
The initial idea of this section was to bring as much clarity as possible to avoid support questions; It was modeled after https://www.telerik.com/kendo-react-ui/pricing/. It's why the h2 is called "What to expect from our available plans". I'm not against addressing some of the "losses" that customers might have on the pricing page (fear or losing out) but I think that it should stay light, https://mui.com/x/ would be better suited for this.

Reply:
I was revisiting this and what you described sounds like a special section in the frequently asked questions. I believe "What to expect from the plans" could focus on "What you get during your license term". 
So maybe "Development license" and "Volume discount" could be moved to the FAQ?

Reply:
I think that these points will get less visibility on the FAQ. I have updated the h2 to "Key information about the paid plans" to encourage people to read it all.